### PR TITLE
Godot environment test

### PR DESCRIPTION
**Pull request by EngineeringDude123**
download the exe to test the basic character movement functionality and the environment

### CODE REVIEW

File: Godot Files/wor7943.tmp
Comment (chanson02) [inline Godot Files/wor7943.tmp:1]: Remove temp file?

File: Godot Files/Stalker.exe
Comment (chanson02) [inline Godot Files/Stalker.exe:1]: Dont track binary files.

File: Godot Files/Stalker.console.exe
Comment (chanson02) [inline Godot Files/Stalker.console.exe:1]: Don't track binary files.

**Review by chanson02 — CHANGES_REQUESTED** [review]

**Comment by chanson02** [comment]
Instead of godot files, what would you think about changing the directory to `src`?

File: Godot Files/wor7943.tmp
Comment (EngineeringDude123) [reply Godot Files/wor7943.tmp:1]: Should be taken care of with the updated gitignore

**Comment by EngineeringDude123** [comment]
> Instead of godot files, what would you think about changing the directory to `src`?

Done

**Pull request merged**